AWSAuthSDK/Sources/AWSAuthUI/AWSSignInViewController.m

```diff
@@ -271,6 +271,14 @@ - (void)setUpUserPoolsUI {
         } else {
             [self.signUpButton removeFromSuperview];
         }
+        
+        // style buttons (tintColor)
+        if (self.config.tintColor) {
+            self.signInButton.backgroundColor = self.config.tintColor;
```

Comment:
1. Why are we setting the background and tint color the same?
1. Why are we using a `tintColor` on a `backgroundColor` property? Do we need to reconsider our naming?
1. We're setting the signInButton background color; why aren't we setting the `forgotPasswordButton` background color?

Reply:
Very good points:

1. the concept of `tintColor` is not accurate. I changed to `primaryColor` since it better communicates the intent. The idea of the `primaryColor` is to allow devs to change the colors of buttons and links.
2. dito #1
3. We are, two lines below the `signInButton`